src/pages/waypoint/api-docs/[[...page]].tsx

```diff
@@ -105,9 +105,13 @@ export async function getStaticProps({ params }) {
 		{ title: 'Waypoint', url: `/waypoint` },
 		{ title: 'API', url: `/waypoint/api-docs` },
 	]
+	// Remove the 'index' page '/waypoint/api-docs' from the breadcrumbs
+	const shouldAddBreadcrumb = !(
+		apiPageProps.operationCategory.slug === productData.slug
+	)
 
 	// Breadcrumbs - Render conditional category
-	if ('operationCategory' in apiPageProps) {
+	if (shouldAddBreadcrumb && 'operationCategory' in apiPageProps) {
```

Comment:
@zchsh lmk if i'm missing context. Seems like waypoint is different from boundary api docs in that everything is just in one index file.

Reply:
Yep, this sounds exactly right! 💯 👍

At some point in the near future we'll probably want to refine things a bit so the differences are handled in a more intentional way, rather than being mostly defined by "works with Boundary by default cause that's what we started with, everything else might need exceptions tacked on" ... I feel like that's boxed us into having to make these kinds of fixes 😅 has felt like a decent skateboard but could use some love to become a .... bicycle or whatever 🤷‍♂️ 🤣